src/pages/ColabPage/ColabPage.tsx

```diff
@@ -31,9 +32,18 @@ export default function ColabPage() {
 
   const [isLoading, setIsLoading] = useState(true)
   const userContext = useContext(IdContext)
+
   if (!userContext) {
```

Comment:
In the future, this should render some kind of error state instead of throwing an error